.github/workflows/cfn-publish-resource.yaml

```diff
@@ -4,7 +4,7 @@ on:
     inputs:
       regions:
         description: "comma seperated AWS regions.Default is all regions."
-        default: "us-east-1,us-west-1,us-west-2,eu-west-1,eu-central-1,ap-southeast-1,ap-southeast-2,ap-northeast-1,ap-northeast-2,sa-east-1,ca-central-1,eu-west-2,eu-west-3,eu-north-1,ap-south-1,ap-east-1,me-south-1,af-south-1,us-east-2,us-west-1,us-west-2,eu-west-1,ap-southeast-1,ap-southeast-2,ap-northeast-1,ap-northeast-2,sa-east-1,ca-central-1,eu-west-2,eu-west-3,eu-north-1,ap-south-1,ap-east-1,me-south-1,af-south-1,us-east-2,ap-south-2,ap-southeast-4,eu-central-2, eu-south-2,me-central-1"
+        default: "af-south-1,ap-east-1,ap-northeast-1,ap-northeast-2,ap-northeast-3,ap-south-1,ap-southeast-1,ap-southeast-2,ap-southeast-3,ca-central-1,eu-central-1,eu-north-1,eu-south-1,eu-west-1,eu-west-2,eu-west-3,me-south-1,sa-east-1,us-east-1,us-east-2,us-west-1,us-west-2,ap-south-2,ap-southeast-4,eu-central-2,eu-south-2,me-central-1"
```

Comment:
I am assuming every time there is a new AWS region this has to be updated? Is there a README.md file related to this process where this is mentioned?

Reply:
Hi marco!! yes this should be update, but still this is the default value, if anyone wants to deploy in a different region they just need to add the region in the deploy field,  There is a readme file for the templates, is not in this pull request because this is just a fix, https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/.github/workflows/templates/README.md